code/__DEFINES/colors.dm

```diff
@@ -352,6 +352,7 @@
 
 /// Color used for default blood
 #define COLOR_BLOOD "#CC0000"
+#define COLOR_BLOOD_VOX "#2299FC"
```

Comment:
mind changing that to a different colour? it's almost identical to preternis blood